key.ui/src/main/java/de/uka/ilkd/key/gui/nodeviews/InnerNodeView.java

```diff
@@ -149,6 +157,28 @@ private Range highlightPos(PosInOccurrence pos, HighlightPainter light)
             // something concerning highlighting does not work in the future, here could
             // be a starting place to find the mistake.
             getHighlighter().addHighlight(r.start() + 1, r.end() + 1, light);
+
+            // scroll the active formula into view
+            SwingUtilities.invokeLater(() -> {
+                try {
+                    ImmutableList<Integer> pathTop =
+                        posTable.pathForPosition(pos.topLevel(), getFilter());
+                    if (pathTop == null) {
+                        return;
+                    }
+                    Range rFormula = posTable.rangeForPath(pathTop);
+                    setCaretPosition(rFormula.start());
+                    Rectangle2D rect = modelToView2D(rFormula.start() + 2);
+                    Rectangle visible = getVisibleRect();
+                    if (rect != null && visible != null
+                            && !visible.contains(rect.getMinX(), rect.getMinY())) {
+                        MainWindow.getInstance().scrollTo((int) rect.getMinY());
+                    }
+                } catch (BadLocationException e) {
+                    // ignore, should never happen
```

Comment:
I also would hope so, but maybe we should log at least ...

Reply:
Done